examples/react-load/src/Home.jsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import { BaseOpenTelemetryComponent } from '@opentelemetry/plugin-react-load';
+
+class Home extends BaseOpenTelemetryComponent {
```

Comment:
Is another example using [hooks/functional components](https://reactjs.org/docs/hooks-intro.html) also possible with this plugin?

Reply:
Related to comment above, but currently my implementation doesn't support functional components